src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/ProgramService.kt

```diff
@@ -9,11 +9,33 @@ import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.repository.ProgramServi
 annotation class ProgramServiceDslMarker
 
 @NomisDataDslMarker
-interface ProgramServiceDsl : CourseActivityDslApi
+interface ProgramServiceDsl {
+  @CourseActivityDslMarker
+  fun courseActivity(
+    courseActivityId: Long = 0,
+    code: String = "CA",
+    prisonId: String = "BXI",
```

Comment:
Changed default prisons to BXI as this is also the default prison on an offender. This means we don't have to include the prison ID in test data setup where it is largely irrelevant wrt what is being tested.